<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fso::fred::dialogs::CampaignEditorDialog</class>
 <widget class="QDialog" name="fso::fred::dialogs::CampaignEditorDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>1023</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Untitled</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="windowLayout">
   <item>
    <widget class="QPushButton" name="btnMenu">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>File...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="mainTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="campaignSpecs">
      <attribute name="title">
       <string>Campaign Specs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="spec1">
         <item>
          <widget class="QLabel" name="lblName">
           <property name="text">
            <string>Campaign Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtName"/>
         </item>
         <item>
          <widget class="QLabel" name="lblType">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtType">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="spec2">
         <item>
          <widget class="QLabel" name="lblDescr">
           <property name="text">
            <string>Campaign Description</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="chkTechReset">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Reset tech db at start of campaign</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txaDescr">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lblShips">
           <property name="text">
            <string>Initially Available Ships</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lblWeapons">
           <property name="text">
            <string>Initially Available Weapons</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QListView" name="lstShips"/>
         </item>
         <item row="1" column="1">
          <widget class="QListView" name="lstWeapons"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="campaignFlow">
      <property name="whatsThis">
       <string/>
      </property>
      <attribute name="title">
       <string>Campaign Flow</string>
      </attribute>
      <layout class="QHBoxLayout" name="campaignGraphLayout" stretch="0,1">
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>11</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="editorControls" stretch="0,0,0,0,0,0,0,0,0,0,1">
         <item>
          <widget class="QLabel" name="lblMissions">
           <property name="text">
            <string>Available Missions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="lstMissions">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoScrollMargin">
            <number>16</number>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="lstMissionDescr">
           <item>
            <widget class="QLabel" name="lblMissionDescr1">
             <property name="text">
              <string>yellow=uneditable/packaged</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblMissionDescr2">
             <property name="text">
              <string>red=not found/loadable, please resolve!</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutLoadMission">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnLoadMission">
             <property name="text">
              <string>Load Mission</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="missionOptions">
           <property name="title">
            <string>Mission Options</string>
           </property>
           <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0">
            <item row="2" column="1">
             <widget class="QLineEdit" name="txtMainhall">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lblMainhall">
              <property name="text">
               <string>Main Hall Name</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="txtDebriefingPersona">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maxLength">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lblBriefingCutscene">
              <property name="text">
               <string>Briefing Cutscene</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="txtBriefingCutscene">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="lblDebriefingPersona">
              <property name="text">
               <string>Debriefing Persona Index</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="missionBranches">
           <property name="title">
            <string>Branches</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QScrollArea" name="scrBranches">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="sexpView">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>415</width>
                 <height>178</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="branchControls">
              <item>
               <widget class="QGroupBox" name="branchOptions">
                <property name="title">
                 <string>Branch Options</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QPushButton" name="btnBranchUp">
                   <property name="text">
                    <string>Move Up</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnBranchDown">
                   <property name="text">
                    <string>Move Down</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnBranchLoop">
                   <property name="text">
                    <string>Toggle Loop</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="spaceBranchOptions">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnRealign">
                <property name="text">
                 <string>Realign Tree</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="txbMissionDescr">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblLoopDescr">
           <property name="text">
            <string>Mission Loop Description</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="txaLoopDescr">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="loopBrief">
           <item row="2" column="1">
            <widget class="QLineEdit" name="txtLoopVoice">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txtLoopAnim">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lblLoopAnim">
             <property name="text">
              <string>Loop brief anim</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblLoopVoice">
             <property name="text">
              <string>Loop brief voice</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btnBrLoopAnim">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="btnBrLoopVoice">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="spaceBottom">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="scrFlowCampaign">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="flowCampaign">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>429</width>
            <height>880</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mainControls">
     <item>
      <widget class="QPushButton" name="btnErrorChecker">
       <property name="text">
        <string>Error Checker</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lstMissions</tabstop>
  <tabstop>txtBriefingCutscene</tabstop>
  <tabstop>txtMainhall</tabstop>
  <tabstop>txtDebriefingPersona</tabstop>
  <tabstop>scrBranches</tabstop>
  <tabstop>btnBranchUp</tabstop>
  <tabstop>btnBranchDown</tabstop>
  <tabstop>btnBranchLoop</tabstop>
  <tabstop>txbMissionDescr</tabstop>
  <tabstop>txaLoopDescr</tabstop>
  <tabstop>txtLoopAnim</tabstop>
  <tabstop>btnBrLoopAnim</tabstop>
  <tabstop>txtLoopVoice</tabstop>
  <tabstop>btnBrLoopVoice</tabstop>
  <tabstop>scrFlowCampaign</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
